factory/etc/nsswitch.conf

```diff
@@ -1,7 +1,7 @@
 # This file is part of systemd.
 
-passwd:         compat mymachines systemd
-group:          compat [SUCCESS=merge] mymachines [SUCCESS=merge] systemd
+passwd:         compat systemd
+group:          compat [SUCCESS=merge] systemd
```

Comment:
This breaks compat in the sense that people who only had `mymachines` before and not `systemd` in those lines will get a broken setup unless they update the config. But I don't see a nice way around this, so I think this is OK to do. But we need to make the NEWS entry very clear about this.